scan.go

```diff
@@ -97,6 +98,26 @@ func (s *ScanService) SearchSource(searchSource *SearchSource) *ScanService {
 	return s
 }
 
+// Source allows the user to set the request body manually without using
+// any of the structs and interfaces in Elastic.
+func (s *ScanService) Source(sourceToSet interface{}) *ScanService {
```

Comment:
I guess we could go with `Source` but I changed that to `BodyString(...)`  and `BodyJson(...)` in other services to be more precise and to relief confusion about the `Source` that sets the body and the `Source` that returns the JSON structure of a DSL object.

But I'm okay with `Source` here because `Scan` will be dead with v5.